src/DDTrace/Tracer.php

```diff
@@ -333,7 +315,7 @@ public function flush()
         if (self::isLogDebugActive()) {
             self::logDebug('Flushing {count} traces, {spanCount} spans', [
                 'count' => $this->getTracesCount(),
-                'spanCount' => $this->getSpanCount(),
+                'spanCount' => dd_trace_closed_spans_count(),
```

Comment:
Does this only count the internally closed spans?

Reply:
It counts both internal and userland spans.